content/en/docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -35,7 +35,7 @@ To enable the rolling update feature of a DaemonSet, you must set its
 `.spec.updateStrategy.type` to `RollingUpdate`.
 
 You may want to set [`.spec.updateStrategy.rollingUpdate.maxUnavailable`](/docs/concepts/workloads/controllers/deployment/#max-unavailable) (default
-to 1) and [`.spec.minReadySeconds`](/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) (default to 0) as well.
+to 1), [`.spec.minReadySeconds`](/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) (default to 0) [`.spec.maxSurge`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-surge) (a beta feature and defaults to 25%) as well.
```

Comment:
```suggestion
is 1), [`.spec.minReadySeconds`](/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) (default is 0), and [`.spec.maxSurge`](/docs/concepts/workloads/controllers/deployment/#max-surge) (default is 25%).
```

Reply:
Hi @ravisantoshgudimetla .
Is there another page or doc section where you could update the feature status (beta)?
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-surge

Reply:
Hi @kbhawkey - I wish to re-use the document available in deployment page instead of creating new page. I will however write a blogpost around this feature in 1.22 timeframe